homeassistant/components/eufylife_ble/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+  "domain": "eufylife_ble",
+  "name": "EufyLife",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/eufylife_ble",
+  "bluetooth": [
+    {
+      "local_name": "eufy T9140"
+    },
+    {
+      "local_name": "eufy T9146"
+    },
+    {
+      "local_name": "eufy T9147"
+    },
+    {
+      "local_name": "eufy T9148"
+    },
+    {
+      "local_name": "eufy T9149"
+    }
+  ],
+  "requirements": ["eufylife_ble_client==0.1.7"],
+  "dependencies": ["bluetooth"],
```

Comment:
```suggestion
  "dependencies": ["bluetooth_adapters"],
```

Reply:
Same question as above. Why is this needed? As far as I can tell, every other Bluetooth integration has `bluetooth` in its dependencies, not `bluetooth_adapters`.

Reply:
https://developers.home-assistant.io/blog/2023/01/12/bluetooth-adapters/

Reply:
Thanks, I had missed that blog post.